<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NadirZenithWidgetClass</class>
 <widget class="QWidget" name="NadirZenithWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>143</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkEnable">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckEnable</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioColor">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>90</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Specific color:</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radioBnGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioAverage">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>64</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Averaged color</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radioBnGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioImage">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>145</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Patch with image:</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">radioBnGroup</string>
   </attribute>
  </widget>
  <widget class="QFrame" name="frameImage">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>166</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bnSelectImage">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>275</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editColor">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>113</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>#999999</string>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>116</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinAngle">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>31</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>3</number>
   </property>
   <property name="value">
    <number>35</number>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>33</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Patch size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>33</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>deg</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkEnable</sender>
   <signal>stateChanged(int)</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAverage</sender>
   <signal>toggled(bool)</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editColor</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAngle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnSelectImage</sender>
   <signal>clicked()</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnSelectImageClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editColor</sender>
   <signal>editingFinished()</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnColorEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editColor</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NadirZenithWidgetClass</receiver>
   <slot>OnColorTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSettingsChanged()</slot>
  <slot>OnSelectImageClicked()</slot>
  <slot>OnColorEditingFinished()</slot>
  <slot>OnColorTextChanged()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="radioBnGroup"/>
 </buttongroups>
</ui>
